src/accountsdb/swiss_map.zig

```diff
@@ -62,6 +62,10 @@ pub fn SwissMapManaged(
             return @call(.always_inline, Unmanaged.remove, .{ &self.unmanaged, key });
```

Comment:
out of scope but `SwissMapManaged` should probs be just `SwissMap` to follow the way zig does it

Reply:
In the future, zig may likely be switching to unmanaged by default, so in this case I decided to just qualify them both to avoid this going out of sync with that at all.

Reply:
makes sense but still not a huge fan of coding for future situations - leads to small things getting out of sync which add up

Reply:
Well the point in this case is that it can't go out of sync, since it's not adhering to any convention, it's simply denoting all the relevant information necessary despite context.
I understand the desire to avoid trying to predict the future, but zig is an evolving language, in which it would be unwise to write code based on patterns & conventions which can go out of sync in the near future, leading to communicating the wrong implicit information.

Reply:
its out of sync with the convention of the rest of the codebase rn

Reply:
As far as I looked, I couldn't really find any other instances where we are implementing any data structures with both a managed and unmanaged version, could you explain how this breaks any currently established convention in the codebase?

Reply:
28f90e9a55d6b8c2d203fe1b612b3f8ca6cc4d1c